### PR TITLE
Disallow multiple

### PR DESCRIPTION
**Pull request by mgoulish**
Disallow multiple connections from same token.

### CODE REVIEW

File: client/connector_create_test.go [hunk 1]
```diff
@@ -237,26 +95,80 @@ func TestSelfConnect(t *testing.T) {
...
 }
 
-func setupTwoNamespaces(t *testing.T, ctx context.Context, tokenCreatorNamespace, tokenUserNamespace string) (tokenCreatorClient, tokenUserClient *VanClient) {
+func TestMultipleConnect(t *testing.T) {
+
+	if !*clusterRun {
+		lightRed := "\033[1;31m"
+		resetColor := "\033[0m"
+		t.Skip(fmt.Sprintf("%sSkipping: This test only works in real clusters.%s", string(lightRed), string(resetColor)))
+		return
```
Comment (nicob87) [inline client/connector_create_test.go:104]: minor comment, this return is not needed, I think ...

File: client/connector_create_test.go [hunk 2]
```diff
@@ -237,26 +95,80 @@ func TestSelfConnect(t *testing.T) {
...
+	assert.Assert(t, err, "Can't make first connection")
+
+	// Try to make a second connection.
+	// This should fail.
+	_, err = userClient.ConnectorCreateFromFile(ctx, secretFileName, types.ConnectorCreateOptions{
+		Name:             "conn2",
+		SkupperNamespace: tokenCreatorNamespace,
+		Cost:             1,
+	})
+	assert.Assert(t, err != nil, "Second connection attempt should fail")
```
Comment (nicob87) [inline client/connector_create_test.go:144]: another minor comment, here you can:
```
assert.Error(t, err, "Already connected to \"token1\".")
```

File: client/connector_create_test.go [hunk 3]
```diff
@@ -237,26 +95,80 @@ func TestSelfConnect(t *testing.T) {
...
+}
+
+func setupTwoNamespaces(t *testing.T, ctx context.Context, tokenCreatorNamespace, tokenUserNamespace string) (tokenCreatorClient, tokenUserClient *VanClient, err error) {
```
Comment (nicob87) [inline client/connector_create_test.go:147]: you can remove "t *testing.T", now it is not used

**Review by nicob87 — APPROVED** [review]
Looks good!

Disallowing the user to make "no sense" things seems to be... "logical"!

**Review by ajssmith — APPROVED** [review]
lgtm

**Pull request merged**